### PR TITLE
Add MVC to persist Bibliography Service configurations

### PR DESCRIPTION
**Pull request by jkeck**
Part of #448 

biblio-service-form

### CODE REVIEW

**Comment by ggeisler** [comment]
@jkeck You guys might have decided this wasn't necessary, or you just haven't gotten to it yet, but I noticed that your screenshot doesn't include the hint text for the Zotero ID field:

> See the API section of the your Zotero library's settings for your user or group ID.

Up to you but wanted to mention it in case in just got lost in the details of the work.

**Comment by jkeck** [comment]
Good :eye: @ggeisler.  Added the help text and updated the screenshot above.

File: app/assets/stylesheets/modules/admin.scss [hunk 1]
```diff
@@ -15,3 +15,11 @@ a[aria-expanded=true] .glyphicon-chevron-right {
 a[aria-expanded=false] .glyphicon-chevron-down {
   display: none;
 }
+
+.text-as-label {
+  font-weight: bold;
+}
+
+.input-like-text {
+  padding-top: 7px;
```
Comment (cbeer) [inline app/assets/stylesheets/modules/admin.scss]: Are there bootstrap variables we could/should have used here?

File: app/controllers/services_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    end
+  end
+
+  private
+
+  def build_resource
+    @bibliography_service = if @exhibit.bibliography_service
+                              @exhibit.bibliography_service
+                            else
+                              ::BibliographyService.new(exhibit_id: @exhibit.id)
```
Comment (cbeer) [inline app/controllers/services_controller.rb]: Could `@exhibit.build_bibliography_service` work here (pushing some of this down into the model, at least)?

File: app/views/services/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <%= link_to t('.zotero_sync.button'), 'javascript:;', class: 'btn btn-default btn-sm' %>
+        <% else %>
+          <p><%= t('.zotero_sync.not_synced') %></p>
+        <% end %>
+      </div>
+    </div>
+
+    <div class="form-actions">
+      <div class="primary-actions">
+        <%= f.submit t('.save'), class: 'btn btn-primary' %>
```
Comment (cbeer) [inline app/views/services/edit.html.erb]: I believe `f.primary` is an alias for `f.submit class: 'btn-primary'` (although I don't know what our existing patterns for this look like..)

File: spec/views/spotlight/shared/_configuration_sidebar.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<h4 class="nav-heading"><%=t(:'spotlight.configuration.sidebar.header') %></h4>
```
Comment (cbeer) [inline spec/views/spotlight/shared/_configuration_sidebar.html.erb]: What is this file being used for?

Comment (jkeck) [inline spec/views/spotlight/shared/_configuration_sidebar.html.erb]: To add the `Services` link below.  FWIW, this is all being merged into a feature branch, so we have time to change direction on this.  This is one thing I was thinking we could do something interesting in Spotlight to accommodate, but our timeframe to complete this work has been severely limited.

Comment (cbeer) [inline spec/views/spotlight/shared/_configuration_sidebar.html.erb]: In `spec` though?

Comment (jkeck) [inline spec/views/spotlight/shared/_configuration_sidebar.html.erb]: Oh.  lol, yeah, we'll kill that. Thanks for that catch.

File: app/views/services/edit.html.erb (hunk 3 above)
Comment (jkeck) [inline app/views/services/edit.html.erb]: I want to say `f.submit` gave me an unstyled button, but I'll check again.

File: app/assets/stylesheets/modules/admin.scss (hunk 1 above)
Comment (mejackreed) [inline app/assets/stylesheets/modules/admin.scss]: Resolved in 9e107bf

File: app/views/services/edit.html.erb (hunk 3 above)
Comment (mejackreed) [inline app/views/services/edit.html.erb]: Resolved in 740ef15

File: spec/views/spotlight/shared/_configuration_sidebar.html.erb (hunk 4 above)
Comment (mejackreed) [inline spec/views/spotlight/shared/_configuration_sidebar.html.erb]: Killed in e39c6df

File: app/controllers/services_controller.rb (hunk 2 above)
Comment (mejackreed) [inline app/controllers/services_controller.rb]: Added in f28dfa7

Comment (mejackreed) [inline app/controllers/services_controller.rb]: Upon further reflection, I felt `find_or_initialize_by` was the behavior we wanted here, and implemented here: e9ec64d

**Pull request merged**